pytest_mock.py

```diff
@@ -10,7 +10,7 @@
 version = '1.2'
 
 
-class MockFixture(object):
+class MockFixture:
```

Comment:
Except that maybe @nicoddemus isn't the only one working on that code :wink:

And I agree about forgetting about old-style things, that's why it should be a new-style class :wink: